certified-connectors/WithoutWire Inventory Platform/apiDefinition.swagger.json

```diff
@@ -41,7 +38,7 @@
                   },
                   "CustomerPoNumber": {
                     "type": "string",
-                    "description": "The customer’s PO Number for the order"
+                    "description": "The customer\u2019s PO Number for the order"
```

Comment:
@PavloNikulinEnavate Unicodes are not allowed in any connector artifact as they may not be supported in all the PA pages.